apps/web/src/components/templates/TestWorkflowModal.tsx

```diff
@@ -5,18 +5,23 @@ import * as Sentry from '@sentry/react';
 import { INotificationTrigger, IUserEntity, INotificationTriggerVariable } from '@novu/shared';
 import { Button, Title, Modal } from '../../design-system';
 import { inputStyles } from '../../design-system/config/inputs.styles';
-import { testTrigger } from '../../api/templates';
 import { useContext, useState } from 'react';
 import { errorMessage, successMessage } from '../../utils/notifications';
 import { AuthContext } from '../../store/authContext';
 import { getSubscriberValue, getPayloadValue } from './TriggerSnippetTabs';
+import { testTrigger } from '../../api/notification-templates';
+import { When } from '../utils/When';
 
 const makeToValue = (subscriberVariables: INotificationTriggerVariable[], currentUser?: IUserEntity) => {
-  const subsVars = getSubscriberValue(
-    subscriberVariables,
-    (variable) =>
+  const subsVars = getSubscriberValue(subscriberVariables, (variable) => {
+    if (variable.name === 'subscriberId' && variable.manual) {
+      return variable.value;
+    }
```

Comment:
At the moment the subscriberId is set through the web client user, i added additional logic in order to pass manual param, so we can reuse this component (trigger button+modal)